metadata.json

```diff
@@ -14,19 +14,19 @@
   "dependencies": [
     {
       "name": "puppetlabs/stdlib",
-      "version_requirement": ">= 4.0.0 < 5.0.0"
+      "version_requirement": ">= 4.25.0 < 6.0.0"
```

Comment:
4.25.0 is needed for `Stdlib::IP::Address`

Reply:
Either that or you can use Stdlib::Compat::IP_address, which was available on earlier versions. 

Not sure how it happened, but this issue leaked into my environment. I had to manually patch this module to get it to work. I'll evaluate updating stdlib to 4.25.0 but it will take me some time, since several of our own modules depend on it.

Reply:
It's not super clear, but I don't think you're supposed to use the compat types generally.
https://github.com/puppetlabs/puppetlabs-stdlib#for-module-developers
> After releasing this version, you can release another breaking change release where you remove all compat types and all calls to validate_legacy.

They were just there as a mechanism for moving away from the old `validate_x` functions.

Reply:
I see, thanks for the clarification!